src/include/storage/checkpoint_io.h

```diff
@@ -0,0 +1,254 @@
+#pragma once
+#include <vector>
+#include "storage/projected_columns.h"
+#include "storage/projected_row.h"
+#include "storage/write_ahead_log/log_io.h"
+
+namespace terrier::storage {
+// TODO(Zhaozhe): Should fetch block_size. Use magic number first.
```

Comment:
No, it does not change over time. Ideally we hope to set it as the size of one disk read/write. The comment here means that maybe we should fetch its size from settings manager.

Reply:
Isn't it possible that this value, since it's set by the SettingsManager, is changed when the database is shut down and restarted? So would you have to account for having to recover from a checkpoint is written in a different block_size than the block_size SettingsManager gave you?